graphql/queryResolvers/moduleCrud.ts

```diff
@@ -0,0 +1,44 @@
+import type { MutationDeleteModuleArgs, MutationAddModuleArgs } from '..'
+import prisma from '../../prisma'
+import { Context } from '../../@types/helpers'
+import { isAdminOrThrow } from '../../helpers/isAdmin'
+import type { Module } from '@prisma/client'
+
+export const modules = (): Promise<Module[]> => {
+  return prisma.module.findMany({
+    include: {
+      author: true,
+      lesson: true
+    }
+  })
+}
+
+export const addModule = async (
+  _parent: void,
+  args: MutationAddModuleArgs,
+  ctx: Context
+): Promise<Module> => {
+  const { req } = ctx
+  await isAdminOrThrow(req)
+  const { content, lessonId, name } = args
+  const authorId = req.user?.id
+  if (!authorId) throw new Error('No User')
+  const module = await prisma.module.create({
```

Comment:
We should catch possible errors and log them. Logs are useful in issue triaging.